buildchain/buildchain/salt_tree.py

```diff
@@ -357,17 +357,16 @@ def _run(self) -> None:
     StaticContainerRegistry(
         root=constants.ISO_IMAGE_ROOT,
         server_root='${}_{}_images'.format(
-            config.PROJECT_NAME.lower(), constants.SHORT_VERSION.replace('.', '_')
+            config.PROJECT_NAME.lower(),
+            constants.VERSION.replace('.', '_').replace('-', '_')
         ),
         name_prefix='{}-{}/'.format(
-            config.PROJECT_NAME.lower(), constants.SHORT_VERSION
+            config.PROJECT_NAME.lower(), constants.VERSION
         ),
         destination=Path(
             constants.ISO_ROOT,
             'salt/metalk8s/repo/files',
-            '99-{}-{}-registry.inc'.format(
```

Comment:
ditto, why removing the 99?

Reply:
Same as above, prefix not needed and we don't need/want to have the version in the path as it's obvious that it's the file of our current version